racket/collects/racket/runtime-path.rkt

```diff
@@ -186,7 +186,8 @@
                                         (path-of
                                          #,(datum->syntax
                                             #'orig-stx
```

Comment:
I guess it doesn’t really matter in this case, since `this-expression-source-file` is a macro and the lexical context on the surrounding syntax object is irrelevant, but it seems slightly suspect to adjust the lexical context here at all. It’d work fine to just use `#'here` for the lexical context and `#'orig-stx` for the source location information.